### ISSUE TITLE
Simple DataScroller does not display items.

### ISSUE THREAD

**Issue by efc** [primefaces/primereact#955]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report
```

**Codesandbox Case (Bug Reports)**

https://codesandbox.io/s/pd5z9

**Current behavior**

A very simple DataScroller shows no items when first rendered.

**Expected behavior**

The DataScroller should show the items corresponding to the array in its `value` property.

**Minimal reproduction of the problem with instructions**

Create a DataScroller with a static value and note that it renders empty. I think most DataScrollers are created with lazy loading values or wait on some API for their data, so this has not been noticed yet. For various reasons, even though my data does come from an API it appears to be static by the time the DataScroller sees it, so this error has become apparent.

Please see https://codesandbox.io/s/pd5z9 for an example of the error.

**Please tell us about your environment:**

macOS 10.14.5
VS Code 1.35.1

* **React version:**

16.8.6

* **PrimeReact version:**

3.1.7

* **Browser:** [all]
 
* **Language:** [TypeScript]

**Comment by efc** [comment]
One clue might be that if I edit `componentDidMount` in `DataScroller.js` so that...

```
if (this.props.lazy) {
    this.load();
}
```

... is replace with simply ...

```
this.load();
```

... then the problem disappears.

Unfortunately, this is not a real fix since other side effects then crop up when the actual data _is_ delayed.

**Comment by mertsincan** [comment]
> One clue might be that if I edit componentDidMount in DataScroller.js so that...
> 
> if (this.props.lazy) {
>     this.load();
> }
> ... is replace with simply ...
> 
> this.load();
> ... then the problem disappears.
- Good catch! This fix'll be resolved the problem.

> Unfortunately, this is not a real fix since other side effects then crop up when the actual data is delayed.
- Could you please attach a sample code? I think that when the data comes to datascroller, its componentDidUpdate method'll update local value in component.

**Comment by efc** [comment]
I did supply sample code in my initial issue description above. It is at https://codesandbox.io/s/pd5z9.

**Comment by efc** [comment]
Also, please note that my "clue" above was not a real fix. I tried this approach and while it fixes my test case at https://codesandbox.io/s/pd5z9 it _breaks_ many other common cases where the data is lazy loaded. So it is not a proper fix and the problem is deeper. Unfortunately, I don't understand the component well enough to debug this, so I have just abandoned the use of `DataScroller` for the time being.

**Comment by efc** [comment]
Oh, I see now that you mean sample code that breaks with the simple fix. I think I tested this with the sample from the PrimeReact documentation and that does _not_ work when this fix is in place.

**Comment by dagmanolis** [comment]
Still no fix on the issue? I am using primereact 3.3.3.
